src/components/AllBooks/index.js

```diff
@@ -1,28 +1,44 @@
-import { Link } from "react-router-dom";
-import { Container, Card, Genre } from "./allBooks.styles";
+import unavailableCover from "../../assets/unavailableCover.png";
+import {
+  Container,
+  Card,
+  Genre,
+  StyledLink,
+  Cover,
+  Title,
+} from "./allBooks.styles";
 
-const AllBooks = (props) => {
-  const mappedBooks = props.data.map((book) => {
+const AllBooks = ({ data, genre }) => {
+  const mappedBooks = data.map((book) => {
+    let imageSrc = "";
+    if (!book.volumeInfo.imageLinks) {
+      imageSrc = unavailableCover;
+    } else {
+      imageSrc = book.volumeInfo.imageLinks.thumbnail;
+    }
```

Comment:
can you use a tern for this?
`const imageSrc = !book.volumeInfo.imageLinks ? unavailableCover : book.volumeInfo.imageLinks.thumbnail`